base_layer/wallet/src/wallet.rs

```diff
@@ -814,7 +819,7 @@ pub fn read_or_create_wallet_type<T: WalletBackend + 'static>(
 pub fn derive_comms_secret_key(master_seed: &CipherSeed) -> Result<CommsSecretKey, WalletError> {
     let comms_key_manager = KeyManager::<PublicKey, KeyDigest>::from(
         master_seed.clone(),
-        KEY_MANAGER_COMMS_SECRET_KEY_BRANCH_KEY.to_string(),
+        TransactionKeyManagerBranch::DataEncryption.get_branch_key(),
```

Comment:
this does not seem correct

Reply:
Leaving a comment for any interested spectators.

*narrator* : It was in fact, not correct.